module_1_task_16/project.py

```diff
@@ -0,0 +1,41 @@
+def triangle_pattern(n):
+    for i in range(1, n + 1):
+        for j in range(1, n + 1):
+            print(j, end=" ")
+        print()
+
+
+def multiplication_table(n):
+    for i in range(1, n + 1):
+        for j in range(1,n + 1):
+            print(f"{i * j : 4}", end=' ')
+        print()
+
+
+def countdown(start):
+    while start > 0:
+        print(start)
+        start = start - 1
+
+
+def main():
```

Comment:
The correct way to write main() in python:
```suggestion
if __name__ == "__main__":
```